src/patternfly/components/Nav/nav.scss

```
@@ -1,132 +1,33 @@
 @import "../../patternfly-utilities";
 
 .pf-c-nav {
-  --pf-c-nav--Width: #{pf-size-prem(290px)};
   --pf-c-nav--Transition: var(--pf-global--Transition);
 
-  // List link
-  --pf-c-nav__list-link--PaddingTop: var(--pf-global--spacer--md);
-  --pf-c-nav__list-link--PaddingRight: var(--pf-global--spacer--md);
-  --pf-c-nav__list-link--md--PaddingRight: var(--pf-global--spacer--xl);
-  --pf-c-nav__list-link--PaddingBottom: var(--pf-global--spacer--sm);
-  --pf-c-nav__list-link--PaddingLeft: var(--pf-global--spacer--md);
-  --pf-c-nav__list-link--md--PaddingLeft: var(--pf-global--spacer--xl);
-
-  // font weight
+  // Nav List link
+  --pf-c-nav__list-link--PaddingY: var(--pf-global--spacer--md);
```

The top/bottom values aren't the same, so this should be

```suggestion
  --pf-c-nav__list-link--PaddingTop: var(--pf-global--spacer--md);
  --pf-c-nav__list-link--PaddingBottom: var(--pf-global--spacer--sm);
```